doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -566,78 +566,104 @@ The query returns all the entities that match all the matching conditions (AND l
+  (if present) represent the path to a sub-property of the *target metadata value* (which should be a
+  JSON object). Such sub-property is defined as the *target property*.
+
+The *target property value* is defined as the value of the JSON property addressed by the list of tokens described
+above i.e. the value of the *target property*.
+
+In case only one token is provided (two in the case of filtering by metadata), then the *target property* will
+be the target NGSI attribute itself (or the target metadata in the case of filtering by metadata) and the
+*target property value* will be the target NGSI attribute value (or the target metadata value in the case
+of filtering by metadata). It is assumed that the value of the target NGSI attribute (or the target metadata
+in the case of filtering by metadata) is not a JSON object in this case.
+
+In the case some of the tokens include `.`, you can use simple quote (`'`) as separator. For example, the following
+attribute path `'a.b'.w.'x.y'` is composed by three tokens: the first token is `a.b`, the second token is `w` and
+the third token is `x.y`.
+
 The list of operators (and the format of the values they use) is as follows:
 
 + **Equal**: `==`. This operator accepts the following types of right-hand side:
-    + Single element, e.g. `temperature==40`. For an entity to match, it must contain an attribute
-      with that exact name (temperature) and value (40).
+    + Single element, e.g. `temperature==40`. For an entity to match, it must contain the target
+      property (temperature) and its value be the query value (40) (or include the value, in the case
+      the target property value is an array).
     + A list of comma-separated values, e.g. `color==black,red`. For an entity to match, it must
-      contain an attribute with that name and the value of the attribute must be **any** of the
-      values in the list (OR clause).
+      contain the target property and its value must be **any** of the values in the list (OR clause)
+      (or include **any** of the values in the list in the case the target property value is an array).
       E.g. entities with an attribute named `color`, whose value is `black` are a match, while
       entities with an attribute named `color` but whose value is `white` do not match.
     + A range, specified as a minimum and a maximum, separated by `..`, e.g. `temperature==10..20`.
-      For an entity to match, it must contain an attribute with that name (temperature), and its
-      value must be in the interval between the minimum and maximum of the range (both included).
-      Ranges can only be used with attributes that represent dates (in ISO8601 format), numbers or
-      strings.
+      For an entity to match, it must contain the target property (temperature),
+      and its value must be in the interval between the minimum and maximum of the range (both included).
```

Comment:
its value must be in the interval between the minimum and maximum of the range =>
its value must be between the upper and lower limits of the range

Except, 'its' should be replaced by "the value of the target property", at 'it' refers to the entity.